components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { ReactComponent as Checked } from '../assets/images/checked.svg'
+
+type Props = {
+  children?: React.ReactNode
+  success?: boolean
+  text?: string
+  title: string
+}
+
+const Card: React.FC<Props> = props => (
+  <div className="row mt-5">
+    <div className="col-sm-5 m-auto" style={{ maxWidth: '604px' }}>
```

Comment:
`maxWidth` -> I don't think we should hardcode a width for a reusable component.

Reply:
I added it to follow  the actual design mockup... I will remove it
<img width="638" alt="Screen Shot 2020-02-08 at 11 09 42 PM" src="https://user-images.githubusercontent.com/25457563/74098036-1eddfa00-4ac8-11ea-91f4-2408fe60dac2.png">